great_expectations_provider/operators/great_expectations.py

```diff
@@ -1,681 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-#
-
-import os
-from datetime import datetime
-from pathlib import Path
-from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Union
-
-import great_expectations as ge
-from airflow.exceptions import AirflowException
-from airflow.hooks.base import BaseHook
-from airflow.models import BaseOperator, BaseOperatorLink, Connection, XCom
-from great_expectations.checkpoint import Checkpoint
-from great_expectations.checkpoint.types.checkpoint_result import CheckpointResult
-from great_expectations.core.batch import (
-    BatchRequest,
-    BatchRequestBase,
-    RuntimeBatchRequest,
-)
-from great_expectations.data_context import BaseDataContext
-from great_expectations.data_context.types.base import (
-    CheckpointConfig,
-    DataContextConfig,
-)
-from great_expectations.data_context.util import instantiate_class_from_config
-from great_expectations.datasource.new_datasource import Datasource
-from great_expectations.util import deep_filter_properties_iterable
-from pandas import DataFrame
-
-if TYPE_CHECKING:
-    from airflow.utils.context import Context
-
-
-class GreatExpectationsDataDocsLink(BaseOperatorLink):
```

Comment:
We would need an intermediate release marking these operators as deprecated and that they will be removed in an upcoming release.

Reply:
We just released 0.3.0 marking these as deprecating. Apologies, this would introduce conflicts on your branch, requesting you to please resolve those conflicts as required for the PR.

Reply:
no problem, was very straightforward - i've now merged the latest.